charmhelpers/contrib/openstack/context.py

```diff
@@ -1567,7 +1569,14 @@ class InternalEndpointContext(OSContextGenerator):
     endpoints.
     """
     def __call__(self):
-        return {'use_internal_endpoints': config('use-internal-endpoints')}
+        ctxt = {'use_internal_endpoints': config('use-internal-endpoints')}
+        rel = os_release('cinder-common', base='icehouse')
```

Comment:
Is the intent that this context is generated on units hosting services which don't have cinder installed, i.e. they make use of cinder endpoints, but won't have cinder-common installed?

If so checking the cinder-common package does not make sense to me.

Reply:
fair enough, i think i'll just have the package passed in via the instantiator or somesuch since im not aware of a generic way to get ost release version.